server/bin/pbench-verify-indexed.py

```diff
@@ -0,0 +1,187 @@
+#!/usr/libexec/platform-python
+# -*- mode: python -*-
+
+"""Pbench Re-Index
+
+Re-index all data for a given date range, "YYYY-MM-DD" to "YYYY-MM-DD".
+
+The process of re-indexing first looks to see which INDEXED, TO-INDEX-TOOL,
+and WONT-INDEX* symlinks exist in the given date range, and moves them to
+TO-RE-INDEX.
+
+NOTE: this interface is intended to be used interactively, this is NOT a
+service that runs as a cronjob.  NO RE-INDEXING STEPS SHOULD BE AUTOMATED
+AT THIS POINT.
+"""
+
+import os
+import re
+import sys
+
+from pathlib import Path
+from argparse import ArgumentParser
+
+from elasticsearch1 import Elasticsearch
+
+import pbench
+
+
+_NAME_ = "pbench-reindex"
+
+tb_pat_r = (
+    r"\S+_(\d\d\d\d)[._-](\d\d)[._-](\d\d)[T_](\d\d)[._:](\d\d)[._:](\d\d)\.tar\.xz"
+)
+tb_pat = re.compile(tb_pat_r)
```

Comment:
At some point we want to compile this once.  Why not here?

If we inline this code, then it will be compiled each time through the loop.

Seems worth one line of code.

Reply:
I think the point is that if we never use `tb_pat_r` after compilation, this might as well be `tb_pat = re.compile(r"...")` 😁

Reply:
Doh!